<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html xmlns:cis="http://zlab.bu.edu/schema/cisml" xmlns:fimo="http://noble.gs.washington.edu/schema/cisml" xmlns:mem="http://noble.gs.washington.edu/meme">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>FIMO Results</title>
<style type="text/css">
        td.left {text-align: left;}
        td.right {text-align: right; padding-right: 1cm;}
      </style>
</head>
<body bgcolor="#D5F0FF">
<a name="top_buttons"></a><hr>
<table summary="buttons" align="left" cellspacing="0"><tr>
<td bgcolor="#00FFFF"><a href="#database_and_motifs"><b>Database and Motifs</b></a></td>
<td bgcolor="#DDFFDD"><a href="#sec_i"><b>High-scoring Motif Occurences</b></a></td>
<td bgcolor="#DDDDFF"><a href="#debugging_information"><b>Debugging Information</b></a></td>
</tr></table>
<br><br><hr>
<center><big><b>FIMO - Motif search tool</b></big></center>
<hr>
<p>
      FIMO version 4.3.0, 
      (Release date: 6 PDT 20)
    </p>
<p>
      For further information on how to interpret these results
      or to get a copy of the FIMO software please access
      <a href="http://meme.nbcr.net">http://meme.nbcr.net</a></p>
<hr>
<center><big><b><a name="database_and_motifs">DATABASE AND MOTIFS</a></b></big></center>
<hr>
<div style="padding-left: 0.75in; line-height: 1em; font-family: monospace;">
<p>
        DATABASE 
        upstream_sequences.fasta<br>
        Database contains 
        9
        sequences,
        900
        residues
      </p>
<p>
        MOTIFS 
        meme_out/meme.html 
        (nucleotide)
        <table>
<thead><tr>
<th style="border-bottom: 1px dashed;">MOTIF</th>
<th style="border-bottom: 1px dashed; padding-left: 1em;">WIDTH</th>
<th style="border-bottom: 1px dashed; padding-left: 1em;" align="left">
              BEST POSSIBLE MATCH
            </th>
</tr></thead>
<tbody>
<tr>
<td align="right">1</td>
<td align="right" style="padding-left: 1em;">13</td>
<td align="left" style="padding-left: 1em;">ATCTGACAAGGAG</td>
</tr>
<tr>
<td align="right">2</td>
<td align="right" style="padding-left: 1em;">6</td>
<td align="left" style="padding-left: 1em;">ACACCA</td>
</tr>
<tr>
<td align="right">3</td>
<td align="right" style="padding-left: 1em;">15</td>
<td align="left" style="padding-left: 1em;">GCATCTGGGCGCCTG</td>
</tr>
</tbody>
</table></p>
<p>
        Random model letter frequencies 
        (from non-redundant database):
        <br>A 0.282 C 0.222 G 0.229 T 0.267 </p>
</div>
<hr>
<center><big><b><a name="sec_i">SECTION I: HIGH-SCORING MOTIF OCCURENCES</a></b></big></center>
<hr>
<ul>
<li> Each of the following 
    10
    motif occurrences has 
    
        p-value less then 
        0.0001</li>
<li> The p-value of a motif occurrence is defined as the
    probability of a random sequence of the same length as the motif
    matching that position of the sequence with as good or better a score.
    </li>
<li> The score for the match of a position in a sequence to a motif
    is computed by by summing the appropriate entries from each column of
    the position-dependent scoring matrix that represents the motif.
    </li>
<li> The q-value of a motif occurrence is defined as the
    false discovery rate if the occurrence is accepted as significant.
    </li>
<li>The table is sorted by increasing p-value.</li>
<li>If the start position is larger then the end position,
    the motif occurrence is on the reverse strand.
    </li>
</ul>
<table border="1">
<thead><tr>
<th>Motif</th>
<th>Sequence Name</th>
<th>Start</th>
<th>End</th>
<th>p-value</th>
<th>q-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left"><a name="occurrences_orf1ab">3</a></td>
<td align="left">orf1ab</td>
<td align="left">73</td>
<td align="left">87</td>
<td align="right">1.34e-09</td>
<td align="right">9.88e-07</td>
</tr>
<tr>
<td align="left"><a name="occurrences_S">3</a></td>
<td align="left">S</td>
<td align="left">69</td>
<td align="left">83</td>
<td align="right">5.51e-09</td>
<td align="right">2.04e-06</td>
</tr>
<tr>
<td align="left"><a name="occurrences_NS7a">1</a></td>
<td align="left">NS7a</td>
<td align="left">11</td>
<td align="left">23</td>
<td align="right">3.07e-08</td>
<td align="right">2.08e-05</td>
</tr>
<tr>
<td align="left"><a name="occurrences_S">1</a></td>
<td align="left">S</td>
<td align="left">23</td>
<td align="left">35</td>
<td align="right">8.52e-06</td>
<td align="right">0.00235</td>
</tr>
<tr>
<td align="left"><a name="occurrences_E">1</a></td>
<td align="left">E</td>
<td align="left">44</td>
<td align="left">56</td>
<td align="right">1.12e-05</td>
<td align="right">0.00235</td>
</tr>
<tr>
<td align="left"><a name="occurrences_M">1</a></td>
<td align="left">M</td>
<td align="left">54</td>
<td align="left">66</td>
<td align="right">1.39e-05</td>
<td align="right">0.00235</td>
</tr>
<tr>
<td align="left"><a name="occurrences_NS6">1</a></td>
<td align="left">NS6</td>
<td align="left">12</td>
<td align="left">24</td>
<td align="right">2.32e-05</td>
<td align="right">0.00312</td>
</tr>
<tr>
<td align="left"><a name="occurrences_orf1ab">1</a></td>
<td align="left">orf1ab</td>
<td align="left">10</td>
<td align="left">22</td>
<td align="right">2.77e-05</td>
<td align="right">0.00312</td>
</tr>
<tr>
<td align="left"><a name="occurrences_NS7c">1</a></td>
<td align="left">NS7c</td>
<td align="left">21</td>
<td align="left">33</td>
<td align="right">3.68e-05</td>
<td align="right">0.00355</td>
</tr>
<tr>
<td align="left"><a name="occurrences_NS7b">1</a></td>
<td align="left">NS7b</td>
<td align="left">79</td>
<td align="left">91</td>
<td align="right">4.75e-05</td>
<td align="right">0.00402</td>
</tr>
</tbody>
</table>
<hr>
<center><big><b><a name="debugging_information">DEBUGGING INFORMATION</a></b></big></center>
<hr>
<p>
    Command line:
    </p>
<pre>fimo --oc . --verbosity 1 --norc meme_out/meme.html upstream_sequences.fasta</pre>
<p>
    Settings:
    </p>
<pre><table>
<tr>
<td style="padding-right: 2em">output directory = .</td>
<td style="padding-left: 5em; padding-right: 2em">MEME file name = meme_out/meme.html</td>
<td style="padding-left: 5em; padding-right: 2em">sequence file name = upstream_sequences.fasta</td>
</tr>
<tr>
<td style="padding-right: 2em">background file name = (null)</td>
<td style="padding-left: 5em; padding-right: 2em">motif name = motif</td>
<td style="padding-left: 5em; padding-right: 2em">allow clobber = true</td>
</tr>
<tr>
<td style="padding-right: 2em">compute q-values = true</td>
<td style="padding-left: 5em; padding-right: 2em">output p-threshold set = false</td>
<td style="padding-left: 5em; padding-right: 2em">output q-threshold set = false</td>
</tr>
<tr>
<td style="padding-right: 2em">text only = false</td>
<td style="padding-left: 5em; padding-right: 2em">scan both strands = false</td>
<td style="padding-left: 5em; padding-right: 2em">max sequence length = 250000000</td>
</tr>
<tr>
<td style="padding-right: 2em">output q-value threshold =   1</td>
<td style="padding-left: 5em; padding-right: 2em">output p-value threshold = 0.0001</td>
<td style="padding-left: 5em; padding-right: 2em">pseudocount = 0.1</td>
</tr>
<tr>
<td style="padding-right: 2em">verbosity = 1</td>
<td style="padding-left: 5em; padding-right: 2em"></td>
<td align="right"></td>
</tr>
</table></pre>
<p>
      This information can be useful in the event you wish to report a
      problem with the FIMO software.
    </p>
<hr>
<span style="background-color: #DDDDFF"><a href="#top_buttons"><b>Go to top</b></a></span>
</body>
</html>
